src/models/tools/table/table-content.test.ts

```diff
@@ -149,6 +149,50 @@ describe("TableContent", () => {
     expect(changeRowValues(change2, 2)).toEqual([3, "y-3"]);
   });
 
+  it.only("can import multi-column authored data with invalid expressions", () => {
```

Comment:
This is the only test that we need to run?

Reply:
Good catch!